### ISSUE TITLE
[Bug] Gradle 8 support

### ISSUE THREAD

**Issue by casper-rinnai** [mono0926/barcode_scan2#73]
https://github.com/flutter/flutter/issues/125181

Flutter app android module can't be upgraded to gradle 8, until all dependencies which use native code are also upgraded or at least have namespace in app/build.gradle.

android {
namespace = "com.example.myapp"
...
}

**Comment by ArocaAlejandro** [comment]
Downgrade your build to gradle 7.

Only need to add the namespace for the module in the package for working with the newest requisites from gradle 8.x. Can do it manually until they fix it entering in the package and adding the namespace="de.mintware.barcode_scan"

**Comment by Letalus** [comment]
@ArocaAlejandro do you have any updates on here?

**Comment by ArocaAlejandro** [comment]
Sorry, i don't see the mail with the response, i've answered on this

#78

**Comment by robinbonnes** [comment]
I've created a [PR](https://github.com/mono0926/barcode_scan2/pull/87), you can use my fork untill this is merged by changing your `pubspec.yaml` to:

```
dependencies:
  barcode_scan2:
    git:
      url: https://github.com/robinbonnes/barcode_scan2
```

**Comment by mono0926** [comment]
Merged #87 and released v4.3.2 🐶
